src/utils/hooks.js

```diff
@@ -15,17 +22,33 @@ export function useEmbed() {
       path += `/county/${countyId}`;
     }
   }
+
+  const iFramePath = `${protocol}//${hostname}/embed/${path}`;
+
   const iFrameCodeSnippet =
     '<iframe ' +
-    `src="${protocol}//${
-      hostname === 'localhost' ? 'localhost:3000' : hostname
-    }/embed/${path}" ` +
+    `src="${iFramePath}" ` +
     'title="CoVid Act Now" ' +
     'width="350" ' +
     'height="700" ' +
     'frameBorder="0" ' +
     'scrolling="no"' +
     '></iframe>';
 
-  return { isEmbed, iFrameCodeSnippet };
+  const jsCodeSnippet =
+    '<div ' +
+    'class="covid-act-now-embed" ' +
+    (id ? `data-state-id="${id}" ` : '') +
+    (countyId ? `data-fips-id="${countyId}" ` : '') +
```

Comment:
This is not accurate, but how our `embed.js` script works currently (we actually pass in county name strings) -- I can make that fix here though

Reply:
I think Mike wants us to use the FIPS id in both of these snippets, if possible.